src/main/java/org/opentripplanner/inspector/vector/stop/GroupStopLayerBuilder.java

```diff
@@ -0,0 +1,43 @@
+package org.opentripplanner.inspector.vector.stop;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+import org.locationtech.jts.geom.Envelope;
+import org.locationtech.jts.geom.Geometry;
+import org.opentripplanner.inspector.vector.LayerBuilder;
+import org.opentripplanner.inspector.vector.LayerParameters;
+import org.opentripplanner.transit.model.site.GroupStop;
+import org.opentripplanner.transit.model.site.StopLocation;
+
+public class GroupStopLayerBuilder extends LayerBuilder<StopLocation> {
+
+  private final List<Geometry> geometries;
+
+  public GroupStopLayerBuilder(
+    LayerParameters layerParameters,
+    Locale locale,
+    Collection<GroupStop> groupStops
+  ) {
+    super(
+      new StopLocationPropertyMapper(locale),
+      layerParameters.name(),
+      layerParameters.expansionFactor()
+    );
+    this.geometries =
```

Comment:
Can you add a comment here that we are precomputing the geometries because there are only a few of them?

Reply:
Done.